package.json

```diff
@@ -52,7 +52,10 @@
             "follow-redirects": ">=1.14.8",
             "tmpl": "^1.0.5",
             "vm2@<3.9.6": "^3.9.6",
-            "minimist": "^1.2.6"
+            "minimist": "^1.2.6",
+            "glob-parent@<5.1.2": ">=5.1.2",
```

Comment:
Please provide an explanation for every single override that you want to add. Explain what the issue is and why it is not possible to use a newer version of dependency directly.
The goal is to have zero overrides. Each override is considered an ugly exception.

Reply:
We have 3 overrides 

1.` glob-parent(3.1.0)`
2.` trim: 0.0.1`
3.`trim-newlines 3.0.1`
Which happens due to the dependency of storybook packages. We are currently using latest(stable) version of storybook available. So without overrides we have folowing issues.

<img width="764" alt="image" src="https://user-images.githubusercontent.com/103586139/189071061-92572d25-3ff3-47c2-bb23-c2e83be3d032.png">

Would it be fine to maintain the issue in image in file [version-overrides.md](https://github.com/SAP/open-ux-tools/blob/main/docs/version-overrides.md)

Reply:
We have 3 overrides 

1.` glob-parent(3.1.0)`
2.` trim: 0.0.1`
3.`trim-newlines 3.0.1`
Which happens due to the dependency of storybook packages. We are currently using latest(stable) version of storybook available. So without overrides we have folowing issues.

<img width="764" alt="image" src="https://user-images.githubusercontent.com/103586139/189071061-92572d25-3ff3-47c2-bb23-c2e83be3d032.png">

Would it be fine to maintain the issue in image in file [version-overrides.md](https://github.com/SAP/open-ux-tools/blob/main/docs/version-overrides.md)

Reply:
sure thanks